crawl-ref/source/dat/species/centaur.yaml

```diff
@@ -7,7 +7,7 @@ aptitudes:
   hp: 1
   mp_mod: 0
   mr: 3
-size: SIZE_LARGE
```

Comment:
when reading this PR I'm sort of vacillating on whether it is better to prefer the c names or some kind of more readable thing, for datifying. One advantage of using C names is that `git grep SIZE_LARGE` works uniformly. But on the other hand, hopefully all enums will become enum classes so the names might change anyways. Not sure what to do...